src/modules/launcher/PowerLauncher/ViewModel/ResultsViewModel.cs

```diff
@@ -253,11 +253,24 @@ public void AddResults(List<Result> newRawResults, CancellationToken ct)
             Results.AddRange(newResults);
         }
 
-        public void Sort()
+        public void Sort(MainViewModel.QueryTuningOptions options)
         {
-            var sorted = Results.OrderByDescending(x => x.Result.Score).ToList();
+            var sorted = Results.OrderByDescending(x => (x.Result.Score + (x.Result.SelectedCount * 5))).ToList();
+
+            sorted = Results.OrderByDescending(x => (x.Result.Metadata.WeightBoost + x.Result.Score + (x.Result.SelectedCount * 5))).ToList();
```

Comment:
why sorting twice in a row? (2lines above)

Reply:
also, put this in else below so it doesn't sort twice when SearchQueryTuningEnabled is true